middle_end/flambda2/simplify/simplify_let_cont_expr.ml

```diff
@@ -683,7 +689,7 @@ let rebuild_single_recursive_handler cont
         RE.Continuation_handler.create
           (UA.are_rebuilding_terms uacc)
           variant_params ~handler ~free_names_of_handler:free_names
-          ~is_exn_handler:false
+          ~is_exn_handler:false ~is_cold:false
```

Comment:
I agree that it's correct right now, but it's going to be a source of bugs if/when that changes and we start using the `is_cold` field of recursive continuations, so I really think we should do the right thing and get the correct `is_cold` info here instead of hardcoding it.